common/src/View/MousePreferencePane.cpp

```diff
@@ -302,7 +304,7 @@ namespace TrenchBroom {
         }
 
         void MousePreferencePane::flyMoveSpeedChanged(const int /* value */) {
-            const auto ratio = m_flyMoveSpeedSlider->ratio();
+            const auto ratio = 2.0f * m_flyMoveSpeedSlider->ratio();
```

Comment:
Maybe factor out this 2.0 limit since it's repeated in 3 places including CameraTool3D

Reply:
Done!